doc/intro.md

```diff
@@ -266,11 +261,11 @@ A Mimblewimble transaction includes the following:
 
 * A set of inputs, that reference and spend a set of previous outputs.
 * A set of new outputs that include:
-  * A value and a blinding factor (which is just a new private key) multiplied on
-  a curve and summed to be `r*G + v*H`.
+  * A blinding factor *r* and a value *v* used for scalar multiplication for the curve 
+  points G,H correspondingly, and subsequently summed to be `r*G + v*H`.
   * A range proof that among other things shows that *v* is non-negative.
-* An transaction fee in cleartext.
-* A signature whose private key is computed by taking the excess value (the sum of all
+* A transaction fee in cleartext.
+* A signature its private key is formed by computing the excess value (the sum of all
```

Comment:
Still reads a little awkward.
Maybe - 
"A signature with private key computed from..."

Reply:
I tried again. The thing is that signature has no private key, so "with"/"its" is inaccurate.
Perhaps having here the details about the excess value doesn't really help.